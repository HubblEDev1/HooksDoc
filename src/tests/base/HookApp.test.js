import {shallow} from 'enzyme';
import { HookApp } from '../../HookApp';

import '../../setupTests';

describe('Testing Hooks', () => {
    test('Testing HookApp', () => {
        const wrapper = shallow(<HookApp/>);

        expect(wrapper).toMatchSnapshot();
    })
})