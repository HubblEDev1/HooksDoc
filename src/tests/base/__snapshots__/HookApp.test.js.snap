// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Hooks Testing HookApp 1`] = `
<div>
  <h1>
    HookApp
  </h1>
</div>
`;
